tests/test_cloudpath_upload_copy.py

```diff
@@ -235,14 +239,18 @@ def test_copytree(rig, tmpdir):
     # cloud dir to cloud dir that exists
     p2 = rig.create_cloud_path("new_dir2")
     (p2 / "existing_file.txt").write_text("asdf")  # ensures p2 exists
-    p.copytree(p2)
+    p.copytree(p2, dirs_exist_ok=True)
     assert assert_mirrored(p2, p, check_no_extra=False)
 
+    # test dirs_exist_ok
+    with pytest.raises(CloudPathFileExistsError):
+        p.copytree(p2, dirs_exist_ok=False)
+
     (p / "new_file.txt").write_text("hello!")  # add file so we can assert mirror
     with pytest.raises(OverwriteNewerCloudError):
-        p.copytree(p2)
+        p.copytree(p2, dirs_exist_ok=True)
 
-    p.copytree(p2, force_overwrite_to_cloud=True)
+    p.copytree(p2, force_overwrite_to_cloud=True, dirs_exist_ok=True)
```

Comment:
Can you add a new test that tests the behavior when `dirs_exist_ok` is False?

Reply:
Yup! That's added [here](https://github.com/drivendataorg/cloudpathlib/blob/25c4ce7faf1610cc9926d0da8b7430b6686dbf3a/tests/test_cloudpath_upload_copy.py#L225) and [here](https://github.com/drivendataorg/cloudpathlib/blob/25c4ce7faf1610cc9926d0da8b7430b6686dbf3a/tests/test_cloudpath_upload_copy.py#L245).

Reply:
Ah, sorry, didn't see those. 👍